webdriver/tests/bidi/input/perform_actions/key_events.py

```diff
@@ -21,9 +21,8 @@
         (Keys.RIGHT, "RIGHT"),
     ],
 )
-async def test_non_printable_key_sends_events(
-    bidi_session, top_context, setup_key_test, key, event
-):
+async def test_non_printable_key_sends_events(bidi_session, top_context, key,
+                                              event):
```

Comment:
Isn't the following better or does the linter complain? We use it quite often.

```suggestion
async def test_non_printable_key_sends_events(
    bidi_session, top_context, key, event
):
```